readthedocs/projects/tasks/utils.py

```diff
@@ -127,10 +128,12 @@ def finish_inactive_builds():
         )
         build.save()
         builds_finished += 1
+        projects_finished.add(build.project.slug)
 
     log.info(
         'Builds marked as "Terminated due inactivity".',
         count=builds_finished,
+        project_slugs=projects_finished,
```

Comment:
Might also be helpful to expose build_pk here too, if it isn't already. Usually in debugging, I'll look up the build_pk first to see the flow through our build process.

Reply:
Yeah, I decided to not log them because I thought they could be _a lot_ of pks. However, I don't think it's the common case...

New Relic for the win! https://onenr.io/08woarZJWjx

In the last 7 days, we have 13 log lines with more than 10 builds terminated due activity. There is one log line with 24. Are we OK dumping all those build pks into a log line? It's not _that much_ as I thought.

Reply:
Roger. We could split this up by project too, but I'm guessing it's fine if the build_pk list, project list, or both, are on the long side.